kodilla-exception/src/main/java/com/kodilla/exception/test/FirstChallenge.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.exception.test;
+
+import java.lang.ArithmeticException;
+
+public class FirstChallenge {
+
+    public double divide(double a, double b) throws ArithmeticException {
+        if(b == 0){
+            throw new ArithmeticException();
+        }
+        return a / b;
+    }
+
+    /**
+     * This main can throw an ArithmeticException!!!
+     * @param args
+     */
+    public static void main(String[] args) {
+
+        FirstChallenge firstChallenge = new FirstChallenge();
+
+        try {
+            double result = firstChallenge.divide(3, 0);
+            System.out.println(result);
+        } catch (ArithmeticException e) {
+            System.out.println("I have found the error - " + e);
+        } finally {
+            System.out.println("I am still looking for errors...");
```

Comment:
Tu znów komunikat nie do końca trafiony. Bardziej na miejscu byłby np. `thank you, goodbye` ;)